Index: tools/stunnel.init.in
===================================================================
--- tools/stunnel.init.in
+++ tools/stunnel.init.in	2010-10-14 06:53:59.000000000 +0200
@@ -10,8 +10,8 @@
 # Short-Description: Start or stop stunnel 4.x (SSL tunnel for network daemons)
 ### END INIT INFO
 
-DEFAULTPIDFILE="/var/run/stunnel.pid"
-DAEMON=@prefix@/bin/stunnel
+DEFAULTPIDFILE="/var/run/stunnel/stunnel.pid"
+DAEMON=@prefix@/usr/bin/stunnel
 NAME=stunnel
 DESC="SSL tunnels"
 FILES="/etc/stunnel/*.conf"
@@ -23,7 +23,7 @@
    if test -f $file; then
      CHROOT=`grep "^chroot" $file|sed "s;.*= *;;"`
      PIDFILE=`grep "^pid" $file|sed "s;.*= *;;"`
-     if [ "$PIDFILE" = "" ]; then
+     if [ -z $PIDFILE ]; then
        PIDFILE=$DEFAULTPIDFILE
      fi
      if test -f $CHROOT/$PIDFILE; then
