diff -Naur stunnel-4.27/configure.ac stunnel-4.27.oden/configure.ac
--- stunnel-4.27/configure.ac	2009-02-26 19:59:05.000000000 +0100
+++ stunnel-4.27.oden/configure.ac	2009-06-21 21:11:48.000000000 +0200
@@ -218,6 +218,33 @@
     ;;
 esac
 
+# Determine library dir name
+AC_MSG_CHECKING([for library dirname])
+case $host in
+*-*-linux*)
+  # Test if the compiler is 64bit
+  echo 'int i;' > conftest.c
+  stunnel_cv_cc_64bit_output=no
+  if $CC -c conftest.c; then
+    case `/usr/bin/file conftest.o` in
+    *"ELF 64"*)
+      stunnel_cv_cc_64bit_output=yes
+      ;;
+    esac
+  fi
+  rm -rf conftest*
+  ;;
+esac
+case $host_cpu:$stunnel_cv_cc_64bit_output in
+powerpc64:yes | s390x:yes | sparc64:yes | x86_64:yes)
+  lib="lib64"
+  ;;
+*:*)
+  lib="lib"
+  ;;
+esac
+AC_MSG_RESULT([$lib])
+
 AC_MSG_NOTICE([**************************************** SSL])
 checkssldir() { :
     if test -f "$1/include/openssl/ssl.h"
@@ -242,7 +269,7 @@
     ],
     [
         # Search default localtions of SSL library
-        for maindir in /usr/local /usr/lib /usr/pkg /opt /usr; do
+        for maindir in /usr/local /usr/$lib /usr/pkg /opt /usr; do
             for dir in $maindir/ssl/fips $maindir/ssl/fips-1.0 \
                     $maindir/ssl $maindir/openssl $maindir; do
                 checkssldir $dir && break 2
@@ -264,7 +291,7 @@
 
 # Add SSL includes and libraries
 CFLAGS="$CFLAGS -I$ssldir/include"
-LIBS="$LIBS -L$ssldir/lib -lssl -lcrypto"
+LIBS="$LIBS -L$ssldir/$lib -lssl -lcrypto"
 
 # Check for obsolete RSAref library
 AC_MSG_CHECKING([for obsolete RSAref library])
