diff --git a/tools/stunnel.conf-sample.in b/tools/stunnel.conf-sample.in
index 1895276..eb1b198 100644
--- a/tools/stunnel.conf-sample.in
+++ b/tools/stunnel.conf-sample.in
@@ -13,6 +13,7 @@
 ; Chroot jail can be escaped if setuid option is not used
 setuid = nobody
 setgid = @DEFAULT_GROUP@
+fips = no
 
 ; PID file is created inside the chroot jail (if enabled)
 ;pid = @prefix@/var/run/stunnel.pid
